backend/routes/user.js

```diff
@@ -0,0 +1,304 @@
+const express = require("express");
+const user = express.Router();
+const { PrismaClient } = require("../generated/prisma");
+const prisma = new PrismaClient();
+// create a user
+user.post("/", async (req, res) => {
+  const { id, email, password } = req.body;
+
+  try {
+    const newUser = await prisma.user.create({
+      data: {
+        email,
+        password,
```

Comment:
please fix: right now passwords are stored as plain text. It’s important to hash passwords before saving them in the DB. You’re importing `bcrypt` in your passport config, which is great! Try using it